### DISCUSSION TITLE
Sync all logs to the server from AttendanceLog Tables in many clients with integer Id (primary key)

### DISCUSSION THREAD

**Discussion by Abdelrahmandev1984** [Mimetis/Dotmim.Sync#549] [Q&A]
Dear,
Thanks a lot for this amazing tool.

I have AttendanceLog table which saving the attendance log for the employees in many sites (client DBs), The primary key of this table is integer identity, because one log in a client db may take Id = 1 and in other client db may take Id = 1 again and when sync that to the server the result will be 1 attendance log with the Id 1 which is not valid, Is there is a way to make this library work on a GUID column with leaving the integer Id as the primary key? I think this is also will be useful in the case we need to merge tables based on a condition not based on the primary key.

**Comment by Mimetis** [comment]
It's working with GUID as primary key, but you have to remove the identity increment on your INT column.
Otherwise, it won't work.

Or, you can manage by yourself the identity values. Take a look here : https://github.com/Mimetis/Dotmim.Sync/discussions/442#discussioncomment-361975
